db/migrate/20230314043615_update_users.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateUsers < ActiveRecord::Migration[7.0]
+  def change
+    change_table :users do |t|
+      t.change :name, :string, default: "", null: false
+      t.change :email, :string, default: "", null: false
```

Comment:
https://github.com/ahr-wtag/online_bus_ticketing_system/pull/2/files#r1152845866

Reply:
removed the default for unnecessary field. Thanks apu for mentioning.